defaults/main.yml

```diff
@@ -39,7 +55,7 @@ sensu_api_password: secret
 sensu_api_uchiwa_path: ''
 sensu_api_timeout: 5000
 sensu_client_config: client.json.j2
-sensu_rabbitmq_config: rabbitmq.json.j2
+sensu_rabbitmq_config: sensu-rabbitmq.json.j2
```

Comment:
Sensu related configuration should have prefix `sensu-` as seen in https://github.com/sensu/sensu-ansible/blob/master/tasks/common.yml#L19.